specs/core/0_beacon-chain.md

```diff
@@ -964,8 +968,6 @@ def validate_indexed_attestation(state: BeaconState, indexed_attestation: Indexe
 
     # Verify no index has custody bit equal to 1 [to be removed in phase 1]
     assert len(bit_1_indices) == 0
-    # Verify max number of indices
-    assert len(bit_0_indices) + len(bit_1_indices) <= MAX_INDICES_PER_ATTESTATION
```

Comment:
Maybe worth keeping this line because `len(bit_0_indices) + len(bit_1_indices)` can be greater than `MAX_INDICES_PER_ATTESTATION`.